src/main/proto/phenopackets/schema/v2/core/interpretation.proto

```diff
@@ -81,7 +81,7 @@ enum TherapeuticActionability {
 message VariantInterpretation {
     AcmgPathogenicityClassification acmg_pathogenicity_classification = 1;
     TherapeuticActionability therapeutic_actionability = 2;
-    Variant variant = 3;
```

Comment:
We were going to keep this as-is and add a new `VrsatileInterpretation` as below.

Reply:
This is the one place where I have to push back. It is not helpful (and potentially detrimental) to allow variation to be specified using a mix of the old `Variant` form and the new VRSATILE / VRS approach. We will of course work to make sure all of your previous use cases can be placed (HGVS / SPDI / VCF) in the VRSATILE framework, but it defeats the purpose of this exercise to have two alternate forms of entering the same information.

@pnrobinson @larrybabb I would like your thoughts here as well.

Reply:
I agree with @ahwagner.